pkg/services/ngalert/ngalert.go

```diff
@@ -189,7 +189,7 @@ func (ng *AlertNG) init() error {
 
 	ng.AlertsRouter = alertsRouter
 
-	evalFactory := eval.NewEvaluatorFactory(ng.Cfg.UnifiedAlerting, ng.DataSourceCache, ng.ExpressionService, ng.pluginsStore)
+	evalFactory := eval.NewEvaluatorFactoryWithMetrics(ng.Cfg.UnifiedAlerting, ng.DataSourceCache, ng.ExpressionService, ng.pluginsStore, ng.Metrics.GetEvalMetrics())
```

Comment:
This is just so CI passes - otherwise it fails because grafana/enterprise doesn't pass enough arguments. It will be deleted as soon as grafana/enterprise is updated.